src/e2e-test/java/io/cdap/cdap/ui/utils/HttpRequestHandler.java

```diff
@@ -29,15 +28,14 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.net.HttpURLConnection;
 import java.net.URL;
 import java.util.Map;
 import javax.annotation.Nullable;
 
 public class HttpRequestHandler {
   public static HttpResponse makeHttpRequest(HttpMethod method, String urlStr,
                                              @Nullable Map<String, String> headers,
-                                             @Nullable Map<String, String> body,
+                                             @Nullable String body,
```

Comment:
Changed this to `String` so that handler wouldn't worry about request body format.
- for http requests routed to `Constants.BASE_URL` we use `application/x-www-form-urlencoded` body format (use Helper.convertMapToString(reqBody)).
- for requests routed to `Constants.BASE_SERVER_URL`, we use json (use GSON.toJson(body)).